src/serialize.ts

```diff
@@ -64,8 +64,15 @@ const boolEncoder = (bool: boolean) => {
 }
 
 const encode = (key: string, value: unknown): Buffer | null => {
+  const tag = parameterToTag[key]
+
+  if (!tag) {
+    console.warn(`Unrecognized object parameter: ${key}`)
```

Comment:
Maybe a console.error here would be appropriate as there was an error recognizing a prop?

Reply:
I did consider that, but I think a warning is apt as an error didn't actually occur, but definitely needs flagging though.